include/klee/Interpreter.h

```diff
@@ -39,6 +39,16 @@ class InterpreterHandler {
   virtual llvm::raw_fd_ostream *openOutputFile(const std::string &filename) = 0;
 
   virtual void incPathsExplored() = 0;
+  virtual void incTotalDepthPathsExploredOnExit(unsigned currentDepth) = 0;
+  virtual void incTotalInstructionsOnExit(unsigned currentInstruction) = 0;
```

Comment:
I suggest renaming to `incInstructionDepthOnExit`, which is more uniform to my earlier comment.